docs/platforms/react-native/tracing/instrumentation/react-navigation.mdx

```diff
@@ -17,23 +17,28 @@ The code snippet below shows how to initialize the instrumentation.
 You can configure the instrumentation by passing an options object to the constructor:
 
 ```javascript
-new Sentry.ReactNavigationInstrumentation({
+Sentry.reactNavigationintegration({
   enableTimeToInitialDisplay: true, // default: false
-  routeChangeTimeoutMs: 1000, // default: 1000
+  routeChangeTimeoutMs: 1_000, // default: 1_000
+  ignoreEmptyBackNavigationTransactions: true, // default: true
 });
 ```
 
 ### routeChangeTimeoutMs
 
-This option specifies how long the instrumentation will wait for the route to mount after a change has been initiated before the transaction is discarded. The default value is 1000ms.
+This option specifies how long the instrumentation will wait for the route to mount after a change has been initiated before the transaction is discarded. The default value is `1_000`.
 
 ### enableTimeToInitialDisplay
 
-This option will enable automatic measuring of the time to initial display for each route. To learn more see [Time to Initial Display](/platforms/react-native/tracing/instrumentation/time-to-display).
+This option will enable automatic measuring of the time to initial display for each route. To learn more see [Time to Initial Display](/platforms/react-native/tracing/instrumentation/time-to-display). The default value is `false`.
+
+### ignoreEmptyBackNavigationTransactions
+
+Does not sample transactions that are from routes that have been seen any more and don't have any spans. This removes a lot of the clutter as most back navigation transactions are now ignored. The default value is `true`.
```

Comment:
```suggestion
This ensures that transactions that are from routes that are no longer being seen and don't have any spans, are not being sampled. This removes a lot of clutter, making it so that most back navigation transactions are now ignored. The default value is `true`.

```

Reply:
```suggestion
This ensures that transactions that are from routes that've been seen and don't have any spans, are not being sampled. This removes a lot of clutter, making it so that most back navigation transactions are now ignored. The default value is `true`.
```

It looks good, but I noticed I had a mistake in the original.